packages/libgpg_error.rb

```diff
@@ -3,18 +3,20 @@
 class Libgpg_error < Autotools
   description 'Libgpg-error is a small library that defines common error values for all GnuPG components.'
   homepage 'https://www.gnupg.org/related_software/libgpg-error/index.html'
-  version '1.50'
+  version '1.51'
   license 'GPL-2 and LGPL-2.1'
   compatibility 'all'
-  source_url 'https://dev.gnupg.org/source/libgpg-error.git'
-  git_hashtag "libgpg-error-#{version}"
+  source_url "https://www.gnupg.org/ftp/gcrypt/libgpg-error/libgpg-error-#{version}.tar.bz2"
+  source_sha256 'be0f1b2db6b93eed55369cdf79f19f72750c8c7c39fc20b577e724545427e6b2'
+  # source_url 'https://dev.gnupg.org/source/libgpg-error.git'
+  # git_hashtag "libgpg-error-#{version}"
```

Comment:
Why not keep the source url and use the git hash for git_hashtag instead?

Reply:
As of several hours ago the tag hadn't been pushed yet, though a release had been cut...

Reply:
(They still haven't tagged it...)

Reply:
(Also, the weird git server they use doesn't allow us to specify a git hash to specify...)

Reply:
Still no 1.51 tag: https://dev.gnupg.org/source/libgpg-error/tags/master/